### DISCUSSION TITLE
Where can I find config file sftpgo.json?

### DISCUSSION THREAD

**Discussion by rnenergy** [drakkan/sftpgo#908] [Q&A]
Hello.
I've installed sftpgo via deb package on deb 11. Standrd installation with no additional manipulations. And I can't find config file.
Where it can be found? Or may be I must to create it manually?
Thanks for answers.

**Comment by drakkan** [comment]
`/etc/sftpgo/sftpgo.json`

**Reply by rnenergy** [reply]
Yea, but there is no files and sftpgo is working.

**Reply by drakkan** [reply]
If you haven't explicitly deleted it the config file is there.. Please post the output of the following commands:

```
sudo ls -la /etc/sftpgo/sftpgo.json
```

```
sudo dpkg -L sftpgo
```

**Reply by rnenergy** [reply]
Sorry i've deleted this installation. And installed new one on Ubuntu but from repo and the conf file is on the right place.
Thank for your quick reply.

**Reply by drakkan** [reply]
> Sorry i've deleted this installation. And installed new one on Ubuntu but from repo and the conf file is on the right place. Thank for your quick reply.

The deb you installed manually is signed and uploaded to the repository, so you basically installed the same deb :roll_eyes:

**Reply by rnenergy** [reply]
There are two devs working on this server -  me and proger. Looks like the proger has deleted all files on /etc/sftpgo/

**Reply by rnenergy** [reply]
Communication problems
